pkg/framework/retry.go

```diff
@@ -0,0 +1,35 @@
+package framework
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
+const maxElapsedTime = 2 * time.Minute
+
+// NewCustomExponentialBackoff returns pointer to a backoff.ExponentialBackOff,
+// initialized with custom values. At the moment, we only override the
+// maxElapsedTime.
+func NewCustomExponentialBackoff() *backoff.ExponentialBackOff {
+	b := &backoff.ExponentialBackOff{
+		InitialInterval:     backoff.DefaultInitialInterval,
+		RandomizationFactor: backoff.DefaultRandomizationFactor,
+		Multiplier:          backoff.DefaultMultiplier,
+		MaxInterval:         backoff.DefaultMaxInterval,
+		MaxElapsedTime:      maxElapsedTime,
+		Clock:               backoff.SystemClock,
+	}
+
+	b.Reset()
+
+	return b
+}
```

Comment:
Is it used anywhere?

Reply:
I was planning to use it from the consumers, using it to set a default value when missing on initialization is much nicer :) Also moved to private.